lib/ZwaveClient.ts

```diff
@@ -3293,6 +3309,46 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 		return valueId
 	}
 
+	/**
+	 * Triggered on node ready or node value changes to map certain values (e.g. batteryLevel) to direct node properties.
+	 * @param node The affected node
+	 * @param valueId The value to be mapped (if undefined, all node values are iterated)
+	 */
+	private _mapNodeValuesToProps(
+		node: Z2MNode,
+		valueId?: TranslatedValueID & { [x: string]: any }
+	) {
+		if (valueId) {
+			if (!mappedCommandClasses.has(valueId.commandClass)) return // Return quickly if command class is not synced to node props
+			for (const k in mappedNodeProps) {
+				const syncedValueId: ValueID = mappedNodeProps[k]
+				if (
+					valueId.commandClass === syncedValueId.commandClass &&
+					valueId.property === syncedValueId.property
+				) {
+					logger.info(
+						'Node ' +
+							node.id +
+							": mapping property '" +
+							k +
+							"' => commandClass=" +
+							valueId.commandClass +
+							' (' +
+							CommandClasses[valueId.commandClass] +
+							"), property='" +
+							valueId.property +
+							"'"
+					)
```

Comment:
Move this to `debug` level and use template string instead of concatenation

Reply:
@ahochsteger, thanks for your feedback and kind words :-).
A quick question about naming:
I'm thinking about changing the naming of the functions that can be defined in the nodes table property definition and would like to get your feedback. Esp. I'd like to get rid of the `Fn` suffix to be more flexible in the future, since I can test for a function anyway. E.g.:
* `valueFn` => `value`
* `sortFn` => `customSort` (named after the `customSort` function, or we may just name it `sort`)
* `infoFn` => `info`
* `formatFn` => `format`

What do you think?

Reply:
I think that yes in some cases instead of just checking if them exists adding a `typeof 'function` check would be more clear. Go for it :)

Reply:
@robertsLando the PR is finished and ready to be merged. :-)

I decided to rename them to `customValue`, `customSort`, `customInfo` and `customFormat` to better communicate what it is and to prevent naming clashes, when they are put into the `headers` object of `VDataTable`.